### PR TITLE
Feature/hooks show

### PR DESCRIPTION
**Pull request by ramiloif**
Show failed hooks ,
Works with new data added to report in PR https://github.com/adamgruber/mochawesome/pull/155

### CODE REVIEW

File: src/components/test/test.jsx [hunk 1]
```diff
@@ -37,7 +37,6 @@ class Test extends React.Component {
   render() {
     const { test, enableCode } = this.props;
     const { uuid, title, speed, duration, pass, fail, pending, skipped, err, code, context } = test;
-
```
Comment (adamgruber) [inline src/components/test/test.jsx]: Nit: leave space

**Comment by adamgruber** [comment]
@ramiloif I made some changes that will affect this PR. You'll need to update the type definitions inside `bin/src/types.js` to include the new properties on `test` and `suite`. Also, when updating the tests, files inside `sample-data` will need to be updated to reflect the new data structure.

**Comment by ramiloif** [comment]
Hi,
- Used isHook in order to show hooks differently 
- Added before and after hooks separation 
   In order to show then before or after the tests , this is associated with the PR on mochawesome 
   https://github.com/adamgruber/mochawesome/pull/157
- Removed the binary files .
- Added tests and changed the sample-data files
Here is how it looks like with the hooks 
screen shot 2017-05-06 at 2 49 51
screen shot 2017-05-06 at 2 49 36

**Comment by adamgruber** [comment]
@ramiloif Thanks for your work on this. I used your PR as a starting point to add the feature. It's now available in v2.2.1

**Pull request closed without merging**